src/session/session.c

```diff
@@ -1052,9 +1053,19 @@ init_remote_session(struct session *ses, enum remote_session_flags *remote_ptr,
 
 	} else if (remote & SES_REMOTE_PROMPT_URL) {
 		dialog_goto_url_open(ses);
-
 	} else if (remote & SES_REMOTE_RELOAD) {
 		reload(ses, CACHE_MODE_FORCE_RELOAD);
+	} else if (remote & SES_REMOTE_SEARCH) {
+		if (! uri)
+			return;
+		if (! ! memcmp(uri->string, "search:", strlen("search:"))) {
```

Comment:
Why !! ?
Isn't logically !!memcmp equals to memcmp?
strlen is calculated in runtime, you can avoid it with appropriate sizeof.
 strncmp would be better here IMO.